### ISSUE TITLE
Add `expansions` field on `globus task show`

### ISSUE THREAD

**Issue by sirosen** [globus/globus-cli#397]
Blocked on API support. This is just another Task field. If it's added to the API, we can start displaying it.

The Hosted CLI notes this as:
```
       expansions
           The number of recursive directories given as input.
```

Probably low priority.

**Comment by sirosen** [comment]
Like #398, closing as a wontfix.
